source/include/FreeRTOS_IPv4_Utils.h

```diff
@@ -49,15 +49,15 @@
 /* FreeRTOS+TCP includes. */
 #include "FreeRTOS_IP.h"
 
-/* Set the MAC-address that belongs to a given IPv4 multi-cast address. */
-void vSetMultiCastIPv4MacAddress( uint32_t ulIPAddress,
-                                  MACAddress_t * pxMACAddress );
-
 /* The first IPv4 length checks at the IP-header level. */
 BaseType_t prvChecksumIPv4Checks( uint8_t * pucEthernetBuffer,
                                   size_t uxBufferLength,
                                   struct xPacketSummary * pxSet );
 
+/* Set the MAC-address that belongs to a given IPv4 multi-cast address. */
+void vSetMultiCastIPv4MacAddress( uint32_t ulIPAddress,
```

Comment:
Again why is this moved? We should ideally not move code around, if not needed.

Reply:
This is moved back as per review comments and changes.

Reply:
Yes but it should be moved to the same place as well, there should not show as a code change is it is as it is.